src/NBitcoin/Script.cs

```diff
@@ -444,7 +444,7 @@ public Script(IEnumerable<byte> data)
 
         public Script(byte[] data, bool compressed)
         {
-            if(!compressed)
```

Comment:
this file has lots of unrelated formatting changes, any chance to undo those (formatting better do in its own PR, especially in such a core class)

Reply:
I will undo this and update the changes, force of habit, I will keep this in mind going forward, thanks.